test/infrastructure/kind/mapper.go

```diff
@@ -63,7 +63,12 @@ const (
 	// of the adoption of CgroupnsMode = "private" when running images.
 	Mode0_20 Mode = "kind 0.20"
 
-	latestMode = Mode0_20
+	// Mode0_21 is a Mode that identifies kind v0.21 and newer; all those versions
+	// are running images in a consistent way, that differs from Mode0_29 because
+	// TODO: Add the differences between Mode0_20 and Mode0_21.
```

Comment:
Open TODO? :-)

If there are no differences we could stay at `Mode0_20 :-)

Reply:
How do we know there are no differences?

(some of them could only show up through slightly flaky tests down the line)

Reply:
I think the differences would be if we have something in CAPD code.

Reply:
Good point :D

Reply:
ah right i meant to come back to this.  

just in this file i should revert this change? or are there further implications for this PR if we do not want to mark `Mode0_21` as latest?

Reply:
Yeah, should be good enouf to revert this lines (66 to 71) and set `Mode0_20` for the new mapper entries :-)

Reply:
pushed up the recommended changes when you have a chance to take another look